datalad/interface/tests/test_results.py

```diff
@@ -0,0 +1,46 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Result utility tests
+
+"""
+
+from datalad.interface.results import (
+    annexjson2result,
+)
+from datalad.distribution.dataset import Dataset
+
+from datalad.tests.utils import (
+    eq_,
+    with_tempfile,
+)
+
+
+@ with_tempfile
```

Comment:
Huh, I didn't know that having a space after `@` was legal syntax, but that seems to work fine.